src/main/java/DateTime.java

```diff
@@ -0,0 +1,16 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+public class DateTime {
+    private LocalDate date;
+//    private LocalTime time;
+
+    public DateTime(String date) {
+        this.date = LocalDate.parse(date);
+    }
+
```

Comment:
I noticed your DateTime class only parses dates. Perhaps that can reflected in the class name?